lib/convection/control/cloud.rb

```diff
@@ -20,45 +20,105 @@ def deck
         @cloudfile.deck
       end
 
-      def converge(to_stack, &block)
-        unless to_stack.nil? || stacks.include?(to_stack)
-          block.call(Model::Event.new(:error, "Stack #{ to_stack } is not defined", :error)) if block
-          return
-        end
+      def stack_groups
+        @cloudfile.stack_groups
+      end
 
+      def create_structure(list)
```

Comment:
im going to look into collecting only the stacks we want instead of iterating over all to find them